CONTRIBUTING.md

```diff
@@ -1,174 +1,163 @@
-	- While we heavily discourage it, we do make exceptions for some bug fixes,
-	typically trivial ones that are easy to approve. *Do not depend on this
-	process* - we reserve the right to close your pull request and manually
-	redirect you to a new or existing issue instead.
-- Avoid cosmetic changes, such as renaming, refactoring, or code formatting - if
-a pull requests attempts to change the coding convention, is centred around the
-reorganisation of code, or generally doesn't focus on stability, functionality
-or testability, it will almost always be rejected. [The Rails maintainers have a
-great explanation on why these changes are not desirable.](https://github.com/rails/rails/pull/13771#issuecomment-32746700)
-	- This includes, but is not limited to, changing the spelling of variable
-	names, changing whitespace and newlines in code, switching between tabs and
-	spaces, renaming internal functions, rearranging function parameters, and
-	regrouping code into different folders.
-- Make sure all your changes follow global and local coding convention. Make
-sure to read our [style guide document](./style-guide.md) and observe how code
-near your working area is written. Consistent code is much easier to maintain,
-and avoids style arguments.
-	- In particular, make sure that variable and function names are written to
-	the same standard as other Fusion code. Naming is where a lot of code falls
-	short, and leads to massive readability problems if left unchecked. We will
-	not hesitate to point these things out in code review.
-- When your pull request is noticed by a project maintainer, it will be subject
-to a code review.
-	- Nobody writes perfect code first time - we're all humans. You will almost
-	certainly be asked to make changes as part of this process, so please be
-	patient with us, and make sure to voice your opinion if you disagree with
-	any suggested changes or rationale so we can make your PR the best it can be!
-- Alternatively, if we decide your pull request doesn't quite align with Fusion,
-then we'll explain in as much detail as possible why we don't want to merge it.
-Don't take this personally - some PRs are fantastic, but just in the wrong place
-or proposed at the wrong time.
-- If your code passes code review, your changes will be approved. They may then
-be merged into the main Fusion codebase at a later time, subject to conditions
-around what we intend to include in different Fusion versions.
-	- Sometimes, we may decide to merge your code, but disable it behind a flag.
-	This is often used to disable features we don't want to make public until
-	some other pull requests are implemented or further design work is done.
-- We always make sure to credit pull requests and their contributors in release
-notes.
\ No newline at end of file
+
+### Are you the first person to suggest it?
+
+- Make sure to search for open issues *and* closed issues - it's possible
+your idea was already suggested and rejected.
```

Comment:
Quite a few comments here are shared between bugs and suggestions. It may be worth making a section in the document like "For all cases:"?

Reply:
I prefer the duplication here in case I change guidance for one process or the other independently of each other. It's the same policy I apply for the docs, so I think this should be fine. Will definitely keep an eye on the level of duplication into the future though and keep this in mind.